src/main/java/io/kestra/plugin/neo4j/Query.java

```diff
@@ -0,0 +1,159 @@
+package io.kestra.plugin.neo4j;
+
+import com.google.common.collect.ImmutableMap;
+import io.kestra.core.models.annotations.PluginProperty;
+import io.kestra.core.models.tasks.RunnableTask;
+import io.kestra.core.runners.RunContext;
+import io.kestra.core.serializers.FileSerde;
+import io.reactivex.BackpressureStrategy;
+import io.reactivex.Flowable;
+import io.reactivex.Single;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+import org.neo4j.driver.Record;
+import org.neo4j.driver.Value;
+import org.neo4j.driver.*;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.net.URI;
+import java.util.AbstractMap;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@Schema(
+    title = "Fetch rows matching the given query"
+)
+public class Query extends AbstractNeo4jConnection implements RunnableTask<Query.Output> {
+    @Schema(
+        title = "The Neo4J query to perform"
+    )
+    @PluginProperty(dynamic = true)
+    private String query;
+    @Schema(
+        title = "Fetch the result"
+    )
+    @Builder.Default
+    private boolean fetch = false;
+
+    @Schema(
+        title = "Fetch one row"
+    )
+    @Builder.Default
+    private boolean fetchOne = false;
+    @Schema(
+        title = "Store to Kestra storage"
+    )
+    @Builder.Default
+    private boolean store = false;
```

Comment:
Maybe one of those 3 may be enabled by default ?
For this kind of stuff, maye it's better to propose an enum "queryType" wether than 3 boolean, this will bings more meanings to the config